src/packages/@library/ui/carousel/Carousel.svelte

```diff
@@ -1,12 +1,13 @@
 <script lang="ts">
+  import type { CarouselDataType } from "@library/ui/carousel/types/carousel-data.type";
   import { Ellipse } from "@library/icons";
```

Comment:
import it using "./types/carousel-data.type", we have restricted library module to use any of the app data.